OriginateThemeTests/OriginateThemeTests.m

```diff
@@ -0,0 +1,39 @@
+//
+//  OriginateThemeTests.m
+//  OriginateThemeTests
+//
+//  Created by Allen Wu on 2016-11-28.
+//  Copyright © 2016 Originate Inc. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+@interface OriginateThemeTests : XCTestCase
+
+@end
+
+@implementation OriginateThemeTests
```

Comment:
It'll be replaced immediately after #15 is merged. I need it here so that it'll succeed in this PR.